features/steps/partcad-cli/commands/version.py

```
@@ -0,0 +1,73 @@
+
+@then('the command should exit with a status code of "{exit_code}"')
+def step_impl(context, exit_code):
+    assert context.result.returncode == int(exit_code)
+
+
+@then("CLI version matches package version")
+def step_impl(context: Context) -> None:
+    import subprocess
+    import partcad
+
+    cli_version = subprocess.check_output(["partcad", "version"], stderr=subprocess.STDOUT).decode()
```

Ruff (0.8.0)

`subprocess` call: check for execution of untrusted input

(S603)